bodhi/client/bindings.py

```diff
@@ -855,6 +856,66 @@ def get_releases(self, **kwargs):
         """
         return self.send_request('releases/', verb='GET', params=kwargs)
 
+    @errorhandled
+    def save_release(self, **kwargs):
+        """
+        Save a new or edited release.
+
+        Args:
+            kwargs (dict): The parameters to send with the request.
+        """
+        res = self.send_request('releases/', verb='POST', auth=True,
+                                data=kwargs)
+
+        if 'errors' in res:
+            self.print_errors(res)
+
+        else:
+            print("Saved release:")
+            self.print_release(res)
+
+        return res
+
+    def print_release(self, release, verbose=False):
```

Comment:
Why remove this function from the client and move it into the bindings? The bindings are intended to be a general purpose Python client for programmatic use, not human use, so they shouldn't be printing things to the terminal (because there might not always be a terminal).